D2211R0.md

```diff
@@ -722,9 +722,9 @@ int main() {
   // Assuming an 'enumerators' reflection facility
   std::for_each(enumerators<Color>(), [](Color c) {
     std::cout << int(c) << " = "
-      << inspect(c){ case RED   => "Red",
-                     case GREEN => "Green",
-                     case BLUE  => "Blue",
+      << inspect(c){ case RED   => "Red";
+                     case GREEN => "Green";
+                     case BLUE  => "Blue";
                    }
       << std::endl;
   });
```

Comment:
Also the lines below https://github.com/mpark/wg21/pull/79/files#diff-dd9f4e4f902ad1d36c6e2a35363e59c32604bf754e9c009f8521fc3a356d3491R735